### PR TITLE
html: calculator

### PR DESCRIPTION
**Pull request by slavidichkov**

### CODE REVIEW

File: src/com/clouway/task6/calculator.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Created by com on 16-2-25.
+ */
+
+ function clean() {
+    document.getElementById("input").innerHTML = "";
```
Comment (tsony-tsonev) [inline src/com/clouway/task6/calculator.js]: duplication

File: src/com/clouway/task6/calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+ function deleteLastChar() {
+    var value = document.getElementById("input").innerHTML;
+    document.getElementById("input").innerHTML = value.slice(0, -1);
+ }
+
+ function add(val) {
+    var currentValue = document.getElementById("input").innerHTML;
+    var s = ['.', '*', '/', '+', '-', '%'];
+    console.log(s.indexOf(currentValue.slice(-1)));
```
Comment (tsony-tsonev) [inline src/com/clouway/task6/calculator.js]: remove log

**Comment by tsony-tsonev** [comment]
I didn't see logic to forbid the usage of '.' in one number more than once

**Comment by slavidichkov** [comment]
```javascript
 var currentValue = document.getElementById("input").innerHTML;
    var s = ['.', '*', '/', '+', '-', '%'];
    if (s.indexOf(currentValue.slice(-1)) < 0) {
        document.getElementById("input").innerHTML = (currentValue + val);
    }
```
if one of these characters is the last character of currentValue it wouldnt be added

**Comment by tsony-tsonev** [comment]
What if I write 3.3.3.4  + 4.4.4

File: src/com/clouway/task6/calculator.js (hunk 1 above)
Comment (slavidichkov) [inline src/com/clouway/task6/calculator.js]: fixed

File: src/com/clouway/task6/calculator.js (hunk 2 above)
Comment (slavidichkov) [inline src/com/clouway/task6/calculator.js]: fixed

**Comment by slavidichkov** [comment]
done

File: src/com/clouway/task6/calculator.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+function compute() {
+    try {
+        setValue("input", eval(getValue("input")));
+    }
+    catch (e) {
+        c('Error')
+    }
+}
+
+function getValue(elementId) {
```
Comment (tsony-tsonev) [inline src/com/clouway/task6/calculator.js]: Just rename these methods to getElementValue , setElementValue and you are good to merge this PR

**Comment by tsony-tsonev** [comment]
LGTM

**Pull request closed without merging**